assets/scss/components/_popover.scss

```diff
@@ -33,6 +33,10 @@ $view-minWidth: 320px;
 
 .popover-container--menu {
 	min-width: $block-4;
+
+	&.visibility--a11yShow {
+		position:absolute;
+	}
```

Comment:
can we added a note in the css maybe as to why we are doing? not 💯 obvious why we are, just a suggestion definitely not a blocker